AEPServices/Sources/ui/fullscreen/FullscreenMessage.swift

```diff
@@ -495,6 +495,19 @@
                 }
             }
         }
+
+        // MARK: - debug methods below are used for testing purposes only
```

Comment:
Can we move all the debug functions to their own debug extension in a separate file?

Reply:
i don't think we can - if we move them to a separate file then they won't be able to expose the underlying private methods (which is what we're trying to achieve to begin with).